components/RuiAccordion/src/RuiAccordion.css.ts

```diff
@@ -37,7 +37,7 @@ export const layout = css`
 	}
 
 	.inner-container {
-		padding: --rui-accordion__padding;
+		padding: --accordion__padding;
```

Comment:
Are we ditching the rui prefix now?

Reply:
yeah I liked removing this after we discussed as it makes the CSS variables more agnostic. Also less characters to type :)